contracts/proto/event/v1/event.proto

```diff
@@ -63,7 +63,11 @@ message NitricEvent {
   // A Unique ID for the Nitric Event
   string id = 1;
   // A content hint for the events payload
-  string payload_type = 2;
+  string type = 2;
   // The payload of the event
-  google.protobuf.Struct payload = 3;
+  oneof data {
+    bytes binary_data = 4;
+    string text_data = 5;
+    google.protobuf.Struct proto_data = 3;
```

Comment:
maybe change this to struct_data or object_data?

Reply:
I took the name from the CloudEvents proto spec for consistency https://github.com/cloudevents/spec/blob/v1.0.1/protobuf-format.md#3-data maybe we should just use this